feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog-event.listener.spec.ts

```diff
@@ -48,6 +50,12 @@ class MockModalService {
   }
 }
 
+const mockCartConfig: CartConfig = {
+  cart: {
+    addToCartFeedback: { feedback: ADDED_TO_CART_FEEDBACK.MODAL },
+  },
+};
+
```

Comment:
There could be coverage about what happens when the feedback config is not modal.